client/client.go

```diff
@@ -56,9 +52,8 @@ func (c *client) apiClient(ctx context.Context) (*graphql.Client, error) {
 		return nil, err
 	}
 
-	return graphql.NewClient(c.session.Endpoint(), oauth2.NewClient(
-		context.WithValue(ctx, oauth2.HTTPClient, c.wraps), oauth2.StaticTokenSource(
```

Comment:
`StaticTokenSource` means that it hardcoded the `bearerToken` into the request. We might as well do it by ourselves.

But the main reason I refactored this is something I shared in our team-slack channel: `golang.org/x/oauth2` has super annoying constraints in its `go.mod` file that I hate.